src/usr/local/buildpack/utils/filesystem.sh

```diff
@@ -44,13 +44,17 @@ function setup_directories () {
 
   mkdir -p "${install_dir}"
   # contains the installed tools
-  mkdir -m 770 "$(get_tools_path)"
+  # shellcheck disable=SC2174
+  mkdir -p -m 770 "$(get_tools_path)"
   # contains env for the installed tools
-  mkdir -m 770 "$(get_env_path)"
+  # shellcheck disable=SC2174
+  mkdir -p -m 770 "$(get_env_path)"
   # contains the latest version of the tools
-  mkdir -m 770 "$(get_version_path)"
+  # shellcheck disable=SC2174
+  mkdir -p -m 770 "$(get_version_path)"
   # contains the wrapper and symlinks for the tools
-  mkdir -m 770 "$(get_bin_path)"
+  # shellcheck disable=SC2174
+  mkdir -p -m 775 "$(get_bin_path)"
```

Comment:
So this only works on tests, as `usr/local/bin` already exist and still has 755?

Reply:
yes, we need to assume that the bin folder in the image already has `775` or we need to make sure to have a docker (=integration) test for it.

So in the end we may need to set up the folder in the `install-buildpack`

Reply:
Let me add an integration test for it and add it to the `install-buildpack` method

Reply:
Ok, let's do it in `install-buildpack`

Reply:
I added an integration test, but locally it fails. The bin path is still `755`.
If it fails in the actions as well, then somewhere this is changed back.